docs/guides/observability/configuration.md

```diff
@@ -0,0 +1,333 @@
+| `observability.flows` | Flow execution | Flow started, node execution |
+| `observability.authorization` | Authorization decisions | Policy evaluation |
+
+**Category Filtering Example:**
+```yaml
+observability:
+  enabled: true
+  output:
+    console:
+      enabled: true
+      categories: ["observability.authentication"]  # Only auth events
+    
+    opentelemetry:
+      enabled: true
+      categories: ["observability.flows"]          # Only flow events
+    
+    file:
+      enabled: true
+      categories: []                               # All events
+```
+
+## Common Configurations
+
+### Development Environment
+```yaml
+observability:
+  enabled: true
+  output:
+    console:
+      enabled: true
+      format: json
+      categories: ["observability.all"]
+    
+    opentelemetry:
+      enabled: true
+      exporter_type: "stdout"
+      service_name: "thunder-dev"
+      environment: "development"
+      sample_rate: 1.0
+  
+  failure_mode: "graceful"
+```
+
+### Production with Jaeger
+```yaml
+observability:
+  enabled: true
+  output:
+    console:
+      enabled: false
+    
+    opentelemetry:
+      enabled: true
+      exporter_type: "otlp"
+      otlp_endpoint: "jaeger-collector.monitoring.svc:4317"
+      service_name: "thunder-iam"
+      service_version: "1.2.0"
+      environment: "production"
+      sample_rate: 0.05                    # 5% sampling for production load
+      insecure: false
+      categories: ["observability.all"]
+    
+    file:
+      enabled: true
+      file_path: "/var/log/thunder/analytics/observability.log"
+      format: json
+      categories: ["observability.authentication", "observability.authorization"]
+  
+  failure_mode: "graceful"
+```
+
+### Staging with Tempo
+```yaml
+observability:
+  enabled: true
+  output:
+    console:
+      enabled: true
+      format: text
+      categories: ["observability.all"]
+    
+    opentelemetry:
+      enabled: true
+      exporter_type: "otlp"
+      otlp_endpoint: "tempo.staging.internal:4317"
+      service_name: "thunder-iam-staging"
+      service_version: "1.2.0-rc1"
+      environment: "staging"
+      sample_rate: 0.5                     # 50% sampling
+      insecure: true
+      categories: ["observability.all"]
+  
+  failure_mode: "graceful"
+```
+
+### Disabled (Minimal Overhead)
+```yaml
+observability:
+  enabled: false
+```
+
+## Setting Up OpenTelemetry Backends
+
+### Jaeger (Docker Compose)
+```yaml
+services:
+  jaeger:
+    image: jaegertracing/all-in-one:latest
+    ports:
+      - "16686:16686"  # Jaeger UI
+      - "4317:4317"    # OTLP gRPC
+    environment:
+      - COLLECTOR_OTLP_ENABLED=true
+```
+
+Then configure Thunder:
+```yaml
+observability:
+  output:
+    opentelemetry:
+      enabled: true
+      exporter_type: "otlp"
+      otlp_endpoint: "localhost:4317"
+```
+
+Access Jaeger UI at: http://localhost:16686
+
+### Grafana Tempo (Docker Compose)
```

Comment:
same as the concern raised with Jaeger. Can't we use docker commands rather than docker compose

Reply:
This is fixed. Added both docker and docker commands instructions